### ISSUE TITLE
Uncaught TypeError: img.toPng is not a function

### ISSUE THREAD

**Issue by rajesh435** [shwld/mdnote-for-atom#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.28.2 x64
**Electron**: 2.0.5
**OS**: Mac OS X 10.13.6
**Thrown From**: [insert-image-to-atom-project](https://github.com/shwld/insert-image-to-atom-project) package 0.0.13

### Stack Trace

Uncaught TypeError: img.toPng is not a function

```
At /Users/developer/.atom/packages/insert-image-to-atom-project/lib/utils.js:16

TypeError: img.toPng is not a function
    at getClipboardImgBuffer (/packages/insert-image-to-atom-project/lib/utils.js:16:25)
    at PasteClipboardImage.imageSaveAndPaste (/packages/insert-image-to-atom-project/lib/paste-clipboard-image.js:44:23)
    at /packages/insert-image-to-atom-project/lib/insert-image-to-atom-project.js:17:32
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:27:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:156:34)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:360:24)
    at CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:273:23)
    at AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1347:25)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:339:53)
    at emitThree (events.js:136:13)
    at EventEmitter.emit (events.js:217:7)
```

### Commands

```
     -8:12.2.0 core:paste (input.hidden-input)
     -8:10.1.0 core:confirm (input.hidden-input)
     -6:33.6.0 tree-view:paste-selection (div.list-tree.has-collapsable-children.file-tree-scroller.nuclide-file-tree.focusable-panel.tree-view)
     -6:31.9.0 core:confirm (input.hidden-input)
     -6:26.8.0 tree-view:paste-selection (div.list-tree.has-collapsable-children.file-tree-scroller.nuclide-file-tree.focusable-panel.tree-view)
     -6:22.6.0 core:confirm (input.hidden-input)
     -6:04.9.0 tree-view:paste-selection (div.list-tree.has-collapsable-children.file-tree-scroller.nuclide-file-tree.focusable-panel.tree-view)
     -6:03.7.0 core:confirm (input.hidden-input)
     -3:06.1.0 core:paste (div.list-tree.has-collapsable-children.file-tree-scroller.nuclide-file-tree.focusable-panel.tree-view)
     -2:55.5.0 tree-view:paste-selection (div.list-tree.has-collapsable-children.file-tree-scroller.nuclide-file-tree.focusable-panel.tree-view)
     -2:53.8.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
insert-image-to-atom-project 0.0.13 
nuclide 0.323.0 
```

**Comment by shwld** [comment]
Thanks for the feedback!

**Comment by shwld** [comment]
@rajesh435 this issue is closed by v0.4.0. enjoy!
